src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,17 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        if (numbers.isEmpty()) {
+            throw new RuntimeException("Can't get min value from list: " + numbers);
+        }
+        return numbers
+                .stream()
+                .map(string -> string.split(","))
```

Comment:
do not use such naming(string), how to guess what it stores?
```suggestion
                .map(lineNumbers -> lineNumbers.split(SEPARATOR))
```